dont-wreck-my-house/src/main/java/learn/lodging/data/GuestFileRepository.java

```diff
@@ -0,0 +1,130 @@
+package learn.lodging.data;
+
+import learn.lodging.models.Guest;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Repository;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+public class GuestFileRepository implements GuestRepository {
+
+    private final String HEADER = "guest_id,first_name,last_name,email,phone,state";
+    private String filePath;
+
+    public GuestFileRepository(@Value("${guestDataFilePath}")String filePath) {
+        this.filePath = filePath;
+    }
+
+    @Override
+    public List<Guest> findAll(){
+        ArrayList<Guest> result = new ArrayList<>();
+        try (BufferedReader reader = new BufferedReader(new FileReader(filePath))) {
+            reader.readLine();
+
+            for (String line = reader.readLine(); line != null; line = reader.readLine()){
+                String[] fields = line.split(",",-1);
```

Comment:
It would be nice to use a constant for your delimiter. since you use it in more than one place.